scripts/install

```diff
@@ -35,8 +35,12 @@ yunohost service add $app --description="Modern team chat" --log="/var/log/$app/
 #=================================================
 ynh_script_progression "Setup $app..."
 
+# Create symlinks as per https://zulip.readthedocs.io/en/latest/production/ssl-certificates.html#manual-install
+ln -sf "/etc/yunohost/certs/$domain/crt.pem" "/etc/ssl/certs/zulip.combined-chain.crt"
+ln -sf "/etc/yunohost/certs/$domain/key.pem" "/etc/ssl/private/zulip.key"
+
 pushd $install_dir
-	./scripts/setup/install --hostname=$domain --email=$email --certbot --postgresql-version=15 --postgresql-database-name=$db_name --postgresql-database-user=$db_user --postgresql-missing-dictionaries --no-dist-upgrade --no-submit-usage-statistics --push-notifications --agree-to-terms-of-service #--cacert="/etc/yunohost/certs/$domain/crt.pem"
+	./scripts/setup/install --hostname=$domain --email=$email --postgresql-version=15 --postgresql-database-name=$db_name --postgresql-database-user=$db_user --postgresql-missing-dictionaries --no-dist-upgrade --no-submit-usage-statistics --push-notifications --agree-to-terms-of-service
```

Comment:
This script breaks YunoHost: it makes NGINX run as the `zulip` user, and takes over Redis as well. It needs to be analyzed and dismantled in YunoHost-compatible commands.